Gruntfile.js

```diff
@@ -165,7 +165,7 @@ module.exports = function(grunt) {
     replacement: config.tab
   }, {
     // remove default syntax themes AND closing bracket
-    pattern: /\s+\/\* grunt build - remove to end of file(.*(\n|\r))+\}$/m,
+    pattern: /(\s+\/\* grunt build - remove start \*\/)(.*)(\s+\/\* grunt build - remove end \*\/)$/sm,
```

Comment:
These capture groups are unnecessary, right?

Reply:
It's still needed to remove our inline syntax highlighting themes,

Reply:
The capture groups in the regex aren't needed, they were added to help with testing it.

Reply:
Then I'd say remove them.